CHANGELOG.md

```diff
@@ -4,10 +4,29 @@ All notable changes to this project will be documented in this file.
 ## [v4.4.5]
 
 
-## [v4.4.4]
+## [v4.4.4] - 2023-06-13
+
+### Manager
+
+#### Fixed
+
+- The vulnerability scanner stops producing false positives for some Windows 11 vulnerabilities due to a change in the feed's CPE. ([#17178](https://github.com/wazuh/wazuh/pull/17178))
+- Prevented the VirusTotal integration from querying the API when the source alert is missing the MD5. ([#16908](https://github.com/wazuh/wazuh/pull/16908))
+
+### Agent
+
+#### Changed
+
+- The Windows agent package signing certificate has been updated. ([#17506](https://github.com/wazuh/wazuh/pull/17506))
+
+### Ruleset
+
+#### Changed
+
+- Updated all current rule descriptions from "Ossec" to "Wazuh". ([#17211](https://github.com/wazuh/wazuh/pull/17211))
 
 
-## [v4.4.3]
+## [v4.4.3] - 2023-05-26
```

Comment:
Release notes: May25 https://documentation.wazuh.com/current/release-notes/release-4-4-3.html. Tag May26 https://github.com/wazuh/wazuh/releases/tag/v4.4.3.
Pick one

Reply:
The fact that tags should be immutable, documentation should fix the date to May 26